cbmackintosh/overlook
# Establish basic class hierarchy for data model

**Clearly and concisely describe the feature.**
- This PR establishes a functional, modular class hierarch that will serve as a data model for the rest of the project.

**Is this a feature or refactor?**
- Feature. Core functionality

**Problems Encountered & Resulting Solutions**
- None

**Where should the reviewer start?**
- Review class files added to src files.

**How should this be tested?**
- Use console.logs() to verify classes are instantiated correctly upon page load.

**Steps To Take**
- Flesh out the login process more thoroughly. Separate UIs for manager/customer into different files, introduce comphrehensive login pathways before proceeding to build out UIs more thoroughly.

Any questions and possible steps moving forward.
- None
